src/client/driver/iframe-driver.js

```diff
@@ -80,8 +80,11 @@ export default class IframeDriver extends Driver {
                 var inCommandExecution = this.contextStorage.getItem(this.COMMAND_EXECUTING_FLAG) ||
                                          this.contextStorage.getItem(this.EXECUTING_IN_IFRAME_FLAG);
 
-                if (inCommandExecution)
+                if (inCommandExecution) {
```

Comment:
if `inCommandExecution` is `false`, `this.COMMAND_EXECUTING_FLAG` and `this.EXECUTING_IN_IFRAME_FLAG` are `false` both. There is no any reason to set them to `false`